spec/acceptance/mod_passenger_spec.rb

```diff
@@ -193,27 +202,26 @@ class { 'apache::mod::passenger': }
       # even when the passenger process is successfully installed and running
       unless fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemrelease') == '12.04'
         it 'outputs status via passenger-status #General information' do
-          shell('PATH=/usr/bin:$PATH /usr/sbin/passenger-status') do |r|
+          shell('PATH=/usr/bin:$PATH PASSENGER_INSTANCE_REGISTRY_DIR=/var/run /usr/sbin/passenger-status') do |r|
             # spacing may vary
             expect(r.stdout).to match(%r{[\-]+ General information [\-]+})
           end
         end
-        expected_two = if fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemrelease') == '14.04' ||
-                          (fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemrelease') == '16.04') ||
+        expected_two = if fact('operatingsystem') == 'Ubuntu' && ['14.04', '16.04', '18.04'].include?(fact('operatingsystemrelease')) ||
```

Comment:
I think you can drop the Ubuntu version checks here as well.

Reply:
done